### PR TITLE
Implement move tool feature.

### PR DESCRIPTION
**Pull request by joshherkness**

### CODE REVIEW

File: vue/src/components/editor/ToolManager.js [hunk 1]
```diff
@@ -125,6 +180,89 @@ class DeleteTool extends Tool {
...
 
+class MoveTool extends Tool {
+  constructor () {
+    super()
+
+    this.type = 'move'
+    this.keyCode = 51
+    this.icon = 'mdi-cursor-move'
+
+    this.preventRotate = true
```
Comment (kdjohnson) [inline vue/src/components/editor/ToolManager.js:191]: I can still rotate the map.

File: vue/src/components/editor/ToolManager.js [hunk 2]
```diff
@@ -125,6 +180,89 @@ class DeleteTool extends Tool {
   }
 }
 
+class MoveTool extends Tool {
```
Comment (kdjohnson) [inline vue/src/components/editor/ToolManager.js:183]: This code doesn't seem to be working beside the tool icon showing up in the menu.

File: vue/src/components/editor/ToolManager.js [hunk 3]
```diff
@@ -125,6 +180,89 @@ class DeleteTool extends Tool {
...
+    super()
+
+    this.type = 'move'
+    this.keyCode = 51
+    this.icon = 'mdi-cursor-move'
+
+    this.preventRotate = true
+
+    this.data = {
+      model: null
```
Comment (kdjohnson) [inline vue/src/components/editor/ToolManager.js:194]: I get a console error stating this is null.

**Review by kdjohnson — CHANGES_REQUESTED** [review]
Currently this doesn't work for me. This is in Firefox.

**Comment by kdjohnson** [comment]
I get this is in Firefox.
screen shot 2018-03-25 at 10 02 07 pm

**Comment by kdjohnson** [comment]
Testing in Safari and it works fine!

**Comment by kdjohnson** [comment]
Also works in Chrome.

**Comment by joshherkness** [comment]
Hey, If this is deployed, will my maps be deleted?

**Comment by kdjohnson** [comment]
I don't think so. I can just restart the vue service in docker after building the new code and leave the others running. But this does highlight that we are not persisting data for mongo currently as far as Docker is concerned.

**Comment by joshherkness** [comment]
Ya, we need to tackle that.

**Comment by kdjohnson** [comment]
I will look into that.

**Comment by joshherkness** [comment]
Can we merge this in, and create a bug issue to fix the firefox stuff, because I can't do it right now.

**Comment by kdjohnson** [comment]
Yea.

**Review by kdjohnson — APPROVED** [review]

File: vue/src/components/editor/ToolManager.js (hunk 1 above)
Comment (joshherkness) [reply vue/src/components/editor/ToolManager.js:191]: This variable should be removed

**Pull request merged**